src/components/metadata/transform.ts

```diff
@@ -46,7 +47,8 @@ function transformContainer(
     controlType: childControlType,
     [childControlType]: {
       ...transformControlFn(rawContainer[childControlType]),
-      ...(isRoot && { isRootControl: true })
+      ...(isRoot && { isRootControl: true }),
```

Comment:
In this particular case, wouldn't this be equivalent to this:

```suggestion
      isRootControl: isRoot,
```

Reply:
Yes, it is equivalent. The first version only had the isRootControl property that was only relevant for the first control, so it was avoided to add the data to false in all controls.

Now the isPartControl property has been added which is more questionable if it should always be added or not. If isPartControl is always added, I think isRootControl should also be added to make the code more consistent.

If you agree to always add the two properties I make the change.

Reply:
I agree with always adding the properties.